watchtower/src/main.rs

```diff
@@ -244,6 +244,7 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     let mut last_notification_msg = "".into();
     let mut num_consecutive_failures = 0;
     let mut last_success = Instant::now();
+    let mut dedup_key = Hash::new_unique().to_string();
```

Comment:
wdyt about this name instead?
```suggestion
    let mut incident = Hash::new_unique().to_string();
```

Reply:
Yeah dedup_key is what PagerDuty calls it and the only place its called, but since it's defined and rotated generically it would make sense to call it something more generic